src/test/resources/com/tupilabs/testng/parser/issue1/testng-results.xml

```diff
@@ -65,4 +65,67 @@
       </class> <!-- com.tupilabs.test.yahoonews.MyTest -->
     </test> <!-- Default test -->
   </suite> <!-- Default suite -->
+  <suite name="suite2" duration-ms="49" started-at="2013-04-16T22:14:29Z" finished-at="2013-04-16T22:14:29Z">
```

Comment:
You rock @yasassri ! Thanks for such well crafted pull request, and in special for including/updating the tests.